tls/s2n_connection.c

```diff
@@ -222,6 +227,8 @@ int s2n_connection_wipe(struct s2n_connection *conn)
      */
     int mode = conn->mode;
     struct s2n_config *config = conn->config;
+    union s2n_prf_working_space prf_space = conn->prf_space;
```

Comment:
With the new revision, I have added `s2n_signed_evp_digest` and `s2n_evp_digest` structs and am using the `s2n_signed_evp_digest` struct within the `p_hash_state` now. This simplifies copying just the EVP handles, and will also be used to preserve EVP handles in the s2n_hash changes (coming as part of Issue #478).